executor/src/witgen/machines/machine_extractor.rs

```diff
@@ -105,17 +106,13 @@ pub fn split_out_machines<'a, T: FieldElement>(
             log::info!("Detected machine: block");
             machines.push(KnownMachine::BlockMachine(machine));
         } else {
-            log::warn!(
-                "Could not find a matching machine to handle a query to the following witness set:\n{}",
```

Comment:
Where did this go?

Reply:
It got removed, any machines that's not a specialized machine gets instantiated as `Generator`. So you'd just see `Detected machine: vm`.

Reply:
Can you try to give a useful error if we can anticipate that witgen won't work?

Reply:
Do you have a condition in mind? Unlike the other machines, `Generator` does not rely on the existence of some columns or some structure of referenced constant columns. Really the only assumption is that you can compute one row after the other, but I'm not sure how to detect that that's not the case.

Reply:
Ok, then maybe change the message to: "Could not detect a specific machine. Will use the generic VM machine." or something like that